tests/dragonfly/replication_test.py

```diff
@@ -292,7 +292,6 @@ async def test_disconnect_master(df_local_factory, df_seeder_factory, t_master,
     seeder = df_seeder_factory.create(port=master.port, keys=n_keys, dbcount=2)
 
     async def crash_master_fs():
-        await asyncio.sleep(random.random() / 10 + 0.1 * len(replicas))
```

Comment:
I don't quite remember the details, but I did this weird stuff with launching it like this to make it sometimes crash a little later and sometimes earlier. It uncovered more bugs when I was writing replication basics. But maybe its not helpful anymore 🤷🏻

Reply:
I had a random fail with this if the master crashes before all replicas finish handshaking. It's kinda easy to reproduce if you really overload the CPU and the replicas starve. So I split `crash_master_fs()` out of the large await, but adding a random sleep to the crash still sounds helpful I guess :)